fv3core/stencils/a2b_ord4.py

```diff
@@ -25,6 +25,203 @@ def grid():
     return spec.grid
 
 
+@gtscript.function
+def great_circle_dist_noradius(p1a, p1b, p2a, p2b):
```

Comment:
seems like we're not using the with-radius one, can we remove the noradius from the name?

Reply:
sure